routers/web/repo/repo.go

```diff
@@ -590,6 +590,15 @@ func SearchRepo(ctx *context.Context) {
 		return
 	}
 
+	// To improve performance when only the count is requested
+	if ctx.FormBool("count_only") {
+		ctx.SetTotalCountHeader(count)
+		return
+	}
+
+	// To query only what's necessary for the dashboard repo list
+	minimalMode := ctx.FormBool("minimal")
```

Comment:
IIRC, all request to this api is minimal now.

![image](https://user-images.githubusercontent.com/2114189/163507384-609af04c-6e59-4105-a9b4-9e456024e3b5.png)

Reply:
Good catch! https://github.com/go-gitea/gitea/pull/18963/commits/19f811f73b79ebf99b9272c51cd6e7166c87cadd